sdcm/sct_runner.py

```diff
@@ -31,24 +36,17 @@ class SctRunner:
     IMAGE_NAME = f"sct-runner-{VERSION}"
```

Comment:
Let's make this class [abstract](https://docs.python.org/3/library/abc.html)

Reply:
fixed.